### DISCUSSION TITLE
Where search used wallet address?

### DISCUSSION THREAD

**Discussion by growersun** [Chia-Network/chia-blockchain#9021] [Support]
Where search used wallet address? me need, maybe file? how it work? Me need all my addresses with blance, i'm have more than 1500 addreses and after send coin created new addreses? everyday more than 500 transfers.

**Comment by Tomaszu97** [comment]
no idea what exactly did you mean but try chia blockchain explorer site like chiaexplorer.com or xchscan.com

**Comment by boxiaoyizhan** [comment]
我也需要查看所有的地址，请问有找到方法吗？

**Comment by Tomaszu97** [comment]
You know, not everyone on the internet speaks mandarin...

**Comment by loppefaaret** [comment]
> 我也需要查看所有的地址，请问有找到方法吗？
> I also need to check all the addresses. Is there a way to find it?

during the command `chia init` - there is a call that scans the first 500 addresses of any wallet imported on the system, perhaps you can modify something to build a tool that will go even further - but we dont currently have an option for it, as far as im aware

**Reply by growersun** [reply]
and balances too lost? if import mnemonic to new machine? and mee interest list wallets, not shadow scan

**Reply by loppefaaret** [reply]
balances are determined by the transactions stored on the blockchain - you rescanning the blockchain getting a wallet into sync will not remove any coins from the wallet - loss of outgoing history is going to happen, but the balance is kept correct!

**Reply by Oscillator098** [reply]
It would be really good to have such address aggregation.
